settings.py

```
@@ -0,0 +1,14 @@
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "mate-fastapi-city-temperature-management-api"
+    DATABASE_URL: str
```

`DATABASE_URL` is declared without a default, so it is required at runtime. This is acceptable (it causes the app to fail fast when missing) but you must document this required environment variable in the README. Alternatively, consider providing a safe development default such as `sqlite+aiosqlite:///./dev.db` so the app can run without extra env setup for local development.